client/src/components/form/UpdateBiodata.tsx

```
@@ -189,15 +192,19 @@ const MultiStepForm: React.FC = () => {
               className="border border-gray-300 rounded-full p-2 text-xs font-bold text-gray-500 flex items-center"
             >
-              {isFirstStep ? null : <CircleArrowLeft className="w-4 h-4 mr-2" />}
+              {isFirstStep ? null : (
+                <CircleArrowLeft className="w-4 h-4 mr-2" />
+              )}
               {steps[activeStep - 1] ?? 'Initial State'}
             </button>
             <button
               onClick={handleNext}
               className="border border-gray-300 rounded-full p-2 text-xs font-bold text-gray-500 flex items-center"
             >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)